modules/core/arrow-instances-data/src/main/kotlin/arrow/instances/ior.kt

```diff
@@ -12,6 +13,12 @@ interface IorFunctorInstance<L> : Functor<IorPartialOf<L>> {
   override fun <A, B> Kind<IorPartialOf<L>, A>.map(f: (A) -> B): Ior<L, B> = fix().map(f)
 }
 
+@instance(Ior::class)
+interface IorBifunctorInstance : Bifunctor<ForIor> {
+  override fun <A, B, C, D> Kind2<ForIor, A, B>.bimap(fl: (A) -> C, fr: (B) -> D): Kind2<ForIor, C, D> =
+    fix().bimap(fl, fr)
```

Comment:
Ior doesn't implement Bifunctor yet, does it? This may be a stackoverflow.

Reply:
Nevermind, it was already there :)

Reply:
There is a `bimap` method in `Ior`:

```kotlin
inline fun <C, D> bimap(fa: (A) -> C, fb: (B) -> D): Ior<C, D> = fold(
    { Left(fa(it)) },
    { Right(fb(it)) },
    { a, b -> Both(fa(a), fb(b)) }
  )
```